gridcomps/Cap/MAPL_CapGridComp.F90

```diff
@@ -979,12 +979,11 @@ function get_vec_from_config(config, key, rc) result(vec)
 
     cap_import = ""
     if (present) then
-
-       do while(trim(cap_import) /= "::")
+       do while( .true.)
```

Comment:
Hmm. A query for @tclune . As far as my reading of the Standard goes, I think the loop-control bit of do is optional. So this:
```fortran
do
  ...
end do
```
seems legal. 

But is that considered "better" than:
```fortran
do while (.true.)
   ...
end do
```
for readable code?

Reply:
Both are legal.  Modest preference for the simpler form,  but both are rather rare in practice.

But there is a more important problem here.   Why is `cap_import` being set if it is not referenced?   Before it was, but was effectively the same as `.true.` in that reference.       But the real fix should be to eliminate the variable entirely so that people like me don't go look at the loop to see how it is being used/modified there.   The original conditional really screamed that it would get updated in the loop.  The newer form reduces that assumption.  Elimination of the variable makes it crystal clear.

Reply:
Sorry - ignore that last rant.   I got distracted and failed to see that it is actually used in the loop.   Just does not play a role in termination.   However  it does look like the assignment before the loop is unnecessary.

Reply:
I removed the assignment. The name 'cap_import' is also changed to 'value'. This call can be used in any case. At least it was used to read cap_export